src/engine/guidance/post_processing.cpp

```diff
@@ -726,6 +729,13 @@ std::vector<RouteStep> collapseTurns(std::vector<RouteStep> steps)
         return index;
     };
 
+    // TODO needs to check whether lane data is required!
```

Comment:
this looks like a messed up rebase? The function was moved to the top of the file to be accessible from multiple locations